content/en/docs/ops/diagnostic-tools/controlz/index.md

```diff
@@ -28,7 +28,7 @@ To access the ControlZ page of istiod, you can port-forward its ControlZ endpoin
 locally and connect through your local browser:
 
 {{< text bash >}}
-$ istioctl dashboard controlz <istiod pod name> -n istio-system
+$ istioctl dashboard controlz <istiod pod name>.istio-system
```

Comment:
A good question is why doesn't it support `-n`?
cc @esnible

Reply:
I verified this works in _istioctl_ 1.6.12 but is broken in _istioctl_ 1.7.2.  We should fix this there.

However, an improvement for this line, if we fixed the 1.7.x regression, would be `istioctl dashboard controlz deployment/istiod -n istio-system`.  That way the user could just cut-and-paste.

Reply:
That looks reasonable to me.
 
However currently, this does not work with 1.9 beta.

```
[istio-1.9.0-beta.0]$ ./bin/istioctl dashboard controlz deployment/istiod -n istio-system
Error: deployments.apps "istiod" not found
```

So I'm keeping the deployment name until we fix the 1.7 regression.

```
[stio-1.9.0-beta.0]$ ./bin/istioctl dashboard controlz deployment/istiod.istio-system
http://localhost:9876
```